### ISSUE TITLE
Pillow can't read PNG

### ISSUE THREAD

**Issue by peterTorrione** [python-pillow/Pillow#3634]
### What did you do?
Tried to load an image / convert to BMP

### What did you expect to happen?
Load the image / convert to BMP

### What actually happened?
Exception:

<pre>
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Users\...\pytorch_pip\lib\site-packages\PIL\Image.py", line 783, in tobitmap
    self.load()
  File "C:\Users\...\pytorch_pip\lib\site-packages\PIL\ImageFile.py", line 250, in load
    self.load_end()
  File "C:\Users\...\pytorch_pip\lib\site-packages\PIL\PngImagePlugin.py", line 677, in load_end
    self.png.call(cid, pos, length)
  File "C:\Users\...\pytorch_pip\lib\site-packages\PIL\PngImagePlugin.py", line 140, in call
    return getattr(self, "chunk_" + cid.decode('ascii'))(pos, length)
  File "C:\Users\...\pytorch_pip\lib\site-packages\PIL\PngImagePlugin.py", line 356, in chunk_IDAT
    raise EOFError
EOFError
</pre>

### What are your OS, Python and Pillow versions?

* OS: WIndows 10
* Python: 3.7.1
* Pillow: 5.4.0

![2_1_2019 4_55_35 pmperson_04_013](https://user-images.githubusercontent.com/1233805/52316987-08b8cd80-298c-11e9-9f18-f6fa3143271d.png)

Code: 
```
from PIL import Image
s = Image.open("2_1_2019 4_55_35 PMperson_04_013.png")
x = s.tobitmap()

```

**Comment by hugovk** [comment]
Please can you test with the latest 5.4.1? 

It has an IDAT-related and some other PNG fixes:

https://pillow.readthedocs.io/en/stable/releasenotes/5.4.1.html

**Comment by peterTorrione** [comment]
Appears to be fixed in 5.4.1 - 

```
 s = Image.open("..\\2_1_2019 4_55_35 PMperson_04_013.png")
 vals = list(s.getdata())
```

Works as expected!  Thanks!
